modules/internal/DefaultRectangular.chpl

```diff
@@ -151,6 +151,104 @@ module DefaultRectangular {
                ignoreRunning = dataParIgnoreRunningTasks,
                minIndicesPerTask = dataParMinGranularity,
                offset=createTuple(rank, idxType, 0:idxType))
+      where tag == iterKind.standalone && !localeModelHasSublocales {
```

Comment:
This is missing the equivalent of:
```chapel
if chpl__testParFlag then
  chpl__testPar("default rectangular domain follower invoked on ", followThis);
```

I'm not sure if that's important or not (or what it would look like here) I'm just noting it before I foret